### ISSUE TITLE
cc110x: cc110x_state_to_text() is not defined

### ISSUE THREAD

**Issue by hahchenchen** [RIOT-OS/RIOT#8115]
The `cc110x_state_to_text()` is used in cc110x-rxtx.c when enable debug but not defined: https://github.com/RIOT-OS/RIOT/blob/master/drivers/cc110x/cc110x-rxtx.c#L256

So how about remove this function and modify like this:
 `           DEBUG("cc110x: invalid state for sending: %s\n",`
  `                  dev->radio_state==RADIO_RX_BUSY?"RADIO_RX_BUSY":"RADIO_TX_BUSY");`
  `          return -EAGAIN;`
 `   }`

**Comment by PeterKietzmann** [comment]
@hahchenchen what you're saying is that `cc110x_state_to_text` is not implemented, right?  If this is the case, this needs to be fixed. Your proposal only involves send and receive busy states. Are there no other possible states?

**Comment by hahchenchen** [comment]
Yes, there are no other possible  states in this function:`cc110x_send`

**Comment by PeterKietzmann** [comment]
@hahchenchen even no "IDLE" state or something? In that case a solution as proposed seems reasonable. Would you provide a fix?

**Comment by hahchenchen** [comment]
No, i would like to provide a fix for it after i fix the spi issue.

**Comment by maribu** [comment]
@hahchenchen: Does [this merged PR](https://github.com/RIOT-OS/RIOT/pull/9154) fix your SPI issue?

Kind regards,
Marian

**Comment by hahchenchen** [comment]
Yes, it does fix SPI issue.
